### DISCUSSION TITLE
[@types/inquirer] Missing 'checked' property in 'Checkbox' types

### DISCUSSION THREAD

**Discussion by cdugeai** [DefinitelyTyped/DefinitelyTyped#65430] [Issues with a @types package]
When using `checkbox` and try to use `checked` property, we have as typescript error:

```
- error TS2322: Type '{ value: string; checked: true; }' is not assignable to type 'Choice<string>'.
  Object literal may only specify known properties, and 'checked' does not exist in type 'Choice<string>'.

31                 {value: 'Test', checked: true },
                                   ~~~~~~~~~~~~~


Found 1 error in path/to_file.ts:31

```

But when adding `// @ts-ignore` on previous line, it works and pre-selects the option as expected.

#### Versions

My versions are:

```json
  "dependencies": {
    "@inquirer/prompts": "^1.0.1",
  },
  "devDependencies": {
    "@types/inquirer": "^9.0.3",
    "tsc": "^2.0.4",
    "tslint": "^6.1.3",
    "typescript": "^5.0.4"
  }
```

Thanks !

[a comment by an automated account is not shown]

**Comment by neurolag** [comment]
@cdugeai Could you please post the code you're trying to compile?
The `checked` property is part of the type declarations already and it is working:

```ts
import inquirer from "inquirer";

inquirer.prompt(
    [
        {
            type: "checkbox",
            choices: [
                {
                    value: "Test",
                    checked: true
                }
            ]
        }
    ]);
```

See: https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBMAdgRwK7CgUynAZlCEOAIiTQ22IG4AoGs9LKAOjAPBgAoa5e4BtHn2EBvIcIlwYATzCYAXCQDGAC0xKA1gCMIAD2IAacZL6qIwJZgDOiwSftwxD53ABuAQwA2qBSQAq1jCGxi4SquoamAAmijBQPiEuAL6JkgC6qXApwmkAlFRAA
